packages/react-dom/src/events/BrowserEventConstants.js

```diff
@@ -85,7 +86,9 @@ var topLevelTypes = {
     getVendorPrefixedEventName('transitionend') || 'transitionend',
   topVolumeChange: 'volumechange',
   topWaiting: 'waiting',
-  topWheel: 'wheel',
+  topWheel: isEventSupported('wheel')
+    ? 'wheel'
+    : isEventSupported('mousewheel') ? 'mousewheel' : 'DOMMouseScroll',
 };
```

Comment:
Did some quick checks. We have `DOMMouseScroll` for Firefox, but MDN claims that `wheel` is supported as of Firefox 17.0. We support the Firefox Extended Support Release (ESR), which is presently at 52.5.0. I think this is a safe change that will not affect any user. 

Also it also looks like IE9 supports wheel ([though it's not on the window according to MDN, the event is still supported](https://developer.mozilla.org/en-US/docs/Web/Events/wheel)).

So we can drop this check all-together. That would mean we'd revert the changes to this file and just keep the code removal in ReactBrowserEventEmitter.

@aweary, @jquense, or @gaearon any concerns with sending this out in a patch release?

Reply:
I don't mind if your test it in the minimal supported browsers.